### ISSUE TITLE
Support for cargo custom profiles

### ISSUE THREAD

**Issue by marmeladema** [kornelski/cargo-deb#13]
Hello and thank you for maintaining cargo-deb ;)

Recently, cargo custom profiles https://blog.rust-lang.org/2021/12/02/Rust-1.57.0.html#cargo-support-for-custom-profiles have been stabilized and it's now possible to build a crate with `--profile <profile-name>` instead of regular `--release`.

I did not seem to find a way to tell cargo-deb to package using a specific custom profile. Instead specifying `cargo deb -- --profile release-production` lead to the following error:
```
error: conflicting usage of --profile=release-production and --release
The `--release` flag is the same as `--profile=release`.
Remove one flag or the other to continue.
cargo-deb: build failed
```

Is there a workaround at the moment? Right now I am creating a symbolic link release -> release-production.

Thank you for your time :)

**Comment by d2r** [comment]
In the newest release v1.36.0, it appears the `release` profile is [hard-coded](https://github.com/kornelski/cargo-deb/blob/v1.36.0/src/lib.rs#L96).

However, it appears support has been merged to the main branch with #14.

As of now, it does not appear there is a workaround.

**Comment by kornelski** [comment]
Released
